gfw_pixetl/utils/path.py

```diff
@@ -1,25 +1,8 @@
 import errno
 import os
-from typing import Optional
 from urllib.parse import urlparse
 
 
-def get_aws_s3_endpoint(endpoint: Optional[str]) -> Optional[str]:
```

Comment:
I just noticed it's not used anywhere, so is dead code. It came to my attention because the behavior of urlparse subtly changed. Rather than figure out how to fix it I decided to toss it.